snapcraft.yaml

```diff
@@ -851,35 +858,8 @@ parts:
       craftctl default
       sed -i 's#Werror=missing-include-dirs#Wmissing-include-dirs#g' meson.build
 
-  libcanberra:
```

Comment:
I don't see that the package itself is being installed as-is anywhere; I only see the -dev packages for libcanberra and libcanberra-gtk, for building, but not for staging. Also, the libcanberra-pulse package should be added too.